ml-agents/mlagents/trainers/simple_env_manager.py

```diff
@@ -22,7 +19,7 @@ def __init__(self, env: BaseEnv, float_prop_channel: FloatPropertiesChannel):
         super().__init__()
         self.shared_float_properties = float_prop_channel
         self.env = env
-        self.previous_step: EnvironmentStep = EnvironmentStep({}, {}, {})
+        self.previous_step: EnvironmentStep = EnvironmentStep({}, {})
```

Comment:
Should we make a EnvironmentStep.empty()? I think we do this in a few other places?

Reply:
Yes - added